src/main/java/ru/javawebinar/topjava/service/MealService.java

```diff
@@ -0,0 +1,15 @@
+package ru.javawebinar.topjava.service;
+
+import ru.javawebinar.topjava.model.Meal;
+
+import java.util.List;
+
+public interface MealService {
+    public void saveMeal(Meal meal);
+
+    public void removeMeal(Integer id);
+
+    public Meal getMealById(Integer id);
```

Comment:
If it's meal service, it is clear that we working with Meals, so it's redundant to use them twice in method names